examples/client/Locomotion/src/context/user/index.tsx

```diff
@@ -175,11 +175,17 @@ const UserContextProvider = ({ children }: { children: any }) => {
 
   const onLogin = async (phoneNumber: string, channel = 'sms') => {
     const demandSourceId = await AppSettings.getOperationId();
-    await loginApi({
+    const switchDemandSource = Config.SWITCH_DEMAND_SOURCE === 'true';
+    const response = await loginApi({
       phoneNumber,
       channel,
       demandSourceId,
+      switchDemandSource,
     });
```

Comment:
const  { demandSourceId } = response;

Reply:
Now it's selectedDemandSourceId, so no need to rename